pom.xml

```diff
@@ -2554,7 +2554,9 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.imports.ImportControlCheckTest</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.javadoc.WriteTagCheckTest</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.naming.ParameterNameCheckTest</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.regexp.RegexpMultilineCheckTest</param>
```

Comment:
@romani Shouldn't we have a issue for this similar to https://github.com/checkstyle/checkstyle/issues/4605 ?
We shouldn't be pulling in random tests and create new tests specifically for api under it's package.

Reply:
Ideally yes, but it is better to think about it when we come to no ignored lines for pitest